dotnet/src/SemanticKernel.Core/KernelExtensions.cs

```diff
@@ -363,6 +364,42 @@ public static ISKPlugin ImportPluginFromObject(this Kernel kernel, object target
             description));
     #endregion
 
+    #region Kernel.Data helper methods
+
+    private const string KernelCultureDataKey = "kernel.culture";
+
+    /// <summary>
+    /// Associate a culture with the kernel.
+    /// </summary>
+    /// <param name="kernel">The kernel.</param>
+    /// <param name="culture">The culture.</param>
+    public static void SetCulture(this Kernel kernel, CultureInfo culture)
```

Comment:
Why are these extension methods?  Why not just have a `public CultureInfo Culture { get; set; }` on Kernel?

Reply:
Just modeled it through the Data dictionary. The Culture class property is a better alternative.